selfdrive/thermald/thermald.py

```diff
@@ -273,10 +273,10 @@ def thermald_thread():
     # If device is offroad we want to cool down before going onroad
     # since going onroad increases load and can make temps go over 107
     # We only do this if there is a relay that prevents the car from faulting
-    if max_cpu_temp > 107. or bat_temp >= 63. or (has_relay and (started_ts is None) and max_cpu_temp > 70.0):
+    if max_cpu_temp > 107.0 or bat_temp > 62.0 or (has_relay and (started_ts is None) and (max_cpu_temp > 65.0 or bat_temp > 60.)):
```

Comment:
I don't think the 2 degree difference in battery temperature here will have any effect.